stairway/src/main/java/bio/terra/stairway/queue/QueueMessageNoFields.java

```diff
@@ -16,5 +17,7 @@ public void setType(QueueMessageType type) {
   }
 
   @Override
-  public void process(Stairway stairway) {}
+  public boolean process(StairwayImpl stairwayImpl) {
+    return true;
```

Comment:
is this actually a trivial message that should return true, or should this message never actually be used to process? What does it mean for us to have a `QueueMessageNoFields` in the queue?
This does raise the question for me of if `QueueMessage` should even have behavior, or it's subclasses should just be dumb data holders. It seems reasonable for `WorkQueueProcessor` to do the actual `process` logic.

Reply:
The subclasses could be dumb data holders, and the `WorkQueueProcessor` could have a `switch` statement to direct processing. It is over-engineered and I would do it differently if I did it again. But my bias is to leave it be.